core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -1124,4 +1125,12 @@ public synchronized ThriftTransportPool getTransportPool() {
     return thriftTransportPool;
   }
 
+  public synchronized ZookeeperLockChecker getZKLockChecker() {
```

Comment:
When I first saw this i thought it could check any lock (like the manager or gc process) .  It seems limited to just tablet server locks when looking deeper though.  Could make this name reflect this so when someone is looking at client context methods its more apparent w/ further digging. 

```suggestion
  public synchronized ZookeeperLockChecker getTserverLockChecker() {
```

Reply:
Changed in 08892f7